pkg/test/resourcefixture/testdata/basic/logging/v1beta1/logginglogmetric/logmetricprojectref/_generated_object_logmetricprojectref.golden.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogMetric
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 2
+  labels:
+    cnrm-test: "true"
+  name: logginglogmetric-${uniqueId}
+  namespace: ${uniqueId}
+spec:
+  bucketOptions:
+    linearBuckets:
+      numFiniteBuckets: 4
+      offset: 0.5
+      width: 2.5
+  description: An updated sample log metric
+  disabled: true
+  filter: resource.type=gae_app AND severity>=ERROR
+  labelExtractors:
+    hasMass: REGEXP_EXTRACT(jsonPayload.request, ".*([1-9]).*")
+    mass: EXTRACT(jsonPayload.request)
+    sku: EXTRACT(jsonPayload.id)
+  metricDescriptor:
+    displayName: updated-sample-descriptor
+    labels:
+    - description: amount of matter
+      key: mass
+      valueType: STRING
+    - description: whether the item has a mass
+      key: hasMass
+      valueType: BOOL
+    - description: identifying number for item
+      key: sku
+      valueType: INT64
+    launchStage: PRELAUNCH
+    metadata:
+      ingestDelay: 3.5s
+      samplePeriod: 3.5s
+    metricKind: DELTA
+    unit: s
+    valueType: DISTRIBUTION
+  projectRef:
+    kind: Project
```

Comment:
Sorry if I missed the earlier conversation. Are we planning to require "kind" in all reference fields in direct implementation?

Reply:
Oh nevermind, I just saw #1925.

Reply:
Yeah, I'm open to dropping kind if we'd prefer.  This is just a test though, and it reflects what must be done at HEAD today.